common/constants.ts

```diff
@@ -0,0 +1,95 @@
+// majors on which the scraper has been verified to run correctly:
+export const supported2018_2019 = [
```

Comment:
Moved these from scraper tests to common so they can be used for the mapping of supported majors